geode-docs/developing/transactions/about_transactions.html.md.erb

```diff
@@ -22,61 +22,160 @@ limitations under the License.
 <a id="topic_jbt_2y4_wk"></a>
 
 
-This section covers the features of <%=vars.product_name%> transactions.
+This section introduces <%=vars.product_name%> transactions.
+Geode offers an API for client applications that do transactional work.
+While the implementation of the API does not provide
+applications a rigid adherence to all the ACID properties of transactions,
+it handles many situations.
+It is fast in comparison to the slow,
+locking methods of a traditional database.
+
+## Overview of the Application's Transaction
+
+There are two ways that an application can run a transaction:
+
+- The application uses the transaction API.
+Here is a code fragment to show the structure of a basic transaction,
+with its `begin` to start the transaction and `commit` to end the transaction.
+
+    ``` pre
+    CacheTransactionManager txManager =
+              cache.getCacheTransactionManager();
+
+    try {
+        txManager.begin();
+        // ... do work
+        txManager.commit();
+    } catch (CommitConflictException conflict) {
+        // ... do necessary work for a transaction that failed on commit
+    }
+    ```
+
+- Use functions.
+Two approaches combine functions with transactions.
+Section [Transactions and Functions](working_with_transactions.html)
+details the interaction.
+The use of a function can have performance benefits,
+as compared to the application directly using the transaction API.
+The performance benefit results from both the functions
+and the region data residing on servers.
+As the function invokes region operations,
+those operations on region entries stay on the server,
+so there is no network round trip time to do get or put
+operations on region data.
+
+    - A function invocation is embedded in a transaction.
+    The application starts the transaction by invoking `begin`,
+    invokes the function within the transaction,
+    and then does the `commit` to end the transaction.
+    - A transaction is embedded in a function.
```

Comment:
This does not impart performance benefits that we talk about previously, so I don't how much value add this is.